test/Main.hs

```diff
@@ -230,13 +230,14 @@ fromRight (Right b) = b
 fromRight (Left x) = error $ "Right _ was expected but Left " ++ show x ++ " found"
 
 testStyleSheet :: StyleSheet
-testStyleSheet = minimalStyleSheet & styleSheetDxfs .~ [dxf1, dxf2]
+testStyleSheet = minimalStyleSheet & styleSheetDxfs .~ [dxf1, dxf2, dxf3]
                                    & styleSheetNumFmts .~ M.fromList [(164, "0.000")]
                                    & styleSheetCellXfs %~ (++ [cellXf1, cellXf2])
   where
     dxf1 = def & dxfFont ?~ (def & fontBold ?~ True
                                  & fontSize ?~ 12)
     dxf2 = def & dxfFill ?~ (def & fillPattern ?~ (def & fillPatternBgColor ?~ red))
+    dxf3 = def & dxfNumFmt ?~ NumFmt 164 "YYYY/MM/DD"
```

Comment:
I.e. it makes sense to have a shared identifier and reference it in both places and also it would be useful to add a haddock note for `_dxfNumFmt` saying that it seems to be required to be duplicated in `_styleSheetNumFmts` though the spec says nothing explicitly about it

Reply:
Ah, I missed that there was an existing number format used in the test.